### PR TITLE
Changes capulettium to allow for more interactive RP changeling stings

### PR DESCRIPTION
**Pull request by colossusqw**
[LABEL][balance][feature]

## About the PR 
This changes the effect of capu from a KO to a paralysis like effect that also stops you speaking, this won't change its efficacy as a KO chem, just allow whoever got injected it something to look at for the couple of minutes it lasts.

## Why's this needed? 
Changeling stings on RP tend to be a bit of a "you're asleep for 2 minutes now, and then you're dead" kind of thing, and it seems many people on the RP chat don't quite enjoy it, so after some talk with them i decided to make it a bit more interesting for the person who just got ling stung to be able to see what the changeling is doing/whisper things to them, which hopefully will create some more RP.

## Changelog

```changelog
(u)Colossusqw
(+)Makes you aware of your surroundings instead of asleep during the capulettium effect, but still unable to speak or move.
```

### CODE REVIEW

**Comment by colossusqw** [comment]
image
how the status effect that stops you from talking looks

File: code/modules/chemistry/Reagents-Misc.dm [hunk 1]
```diff
@@ -1216,6 +1216,8 @@ datum
 			viscosity = 0.13
 			var/counter = 1
 			var/fakedeathed = 0
+			var/oxygendamage = 0
+
```
Comment (Tarmunora) [inline code/modules/chemistry/Reagents-Misc.dm:1220]:
```suggestion
```

File: code/modules/status_system/statusEffects.dm [hunk 2]
```diff
@@ -1959,6 +1959,15 @@
...
 		return ..(timePassed)
 
+/datum/statusEffect/muted
+	maxDuration = 2 MINUTES
+	id = "muted"
+	name = "Muted"
+	icon_state = "muted"
+	desc = "You don't have the strength to say anything louder than a whisper!"
+	maxDuration = 30 SECONDS
+	var/tickspassed = 0
```
Comment (Tarmunora) [inline code/modules/status_system/statusEffects.dm:1969]:
```suggestion
```

File: code/modules/status_system/statusEffects.dm [hunk 3]
```diff
@@ -1959,6 +1959,15 @@
 			return
 		return ..(timePassed)
 
+/datum/statusEffect/muted
+	maxDuration = 2 MINUTES
```
Comment (Tarmunora) [inline code/modules/status_system/statusEffects.dm:1963]:
```suggestion
```

**Review by Tarmunora — CHANGES_REQUESTED** [review]

**Comment by TobleroneSwordfish** [comment]
As someone who makes way too much capulettium I really like this, any opportunity for villainous monologues is great.

**Comment by MACIEKBAKI** [comment]
You could extend this functionality to the vampire's hypnosis ability as well. It is currently also a full knockout, not allowing for much RP with it honestly.

**Comment by colossusqw** [comment]
> You could extend this functionality to the vampire's hypnosis ability as well. It is currently also a full knockout, not allowing for much RP with it honestly.

oooh that sounds good, if this gets merged i'm gonna PR it too then, thanks for the idea!

**Comment by virvatuli-gh** [comment]
I like it, makes capulettium more interesting beyond just being sleepy time

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**